proto/introspection_events.proto

```diff
@@ -44,25 +44,30 @@ message Event {
 }
 
 message NodeCreated {
-  uint64 node_id = 1;
+  // By default the generated JavaScript code converts all 64bit integer types
+  // to the JS `number` type. A potentially lossy conversion since `number` only
+  // supports integers up to 2^53 - 1. This is remedied via the [jstype =
+  // JS_STRING] annotation. Ref:
+  // https://github.com/protocolbuffers/protobuf/issues/3666
+  uint64 node_id = 1 [jstype = JS_STRING];
```

Comment:
Aside: we could probably get away with only doing this for `Handle` values, as they're the only ones which are randomized, but being consistent across `u64` types makes sense.

Reply:
Yup, that was my thinking. While it's unlikely for the channel or node IDs to ever exceed 2^53 - 1 in practice, it seems safer to stick with this method for all 64 bit integers.